src/crypto/tls/tls_test.go

```diff
@@ -790,7 +790,7 @@ func TestCloneNonFuncFields(t *testing.T) {
 		switch fn := typ.Field(i).Name; fn {
 		case "Rand":
 			f.Set(reflect.ValueOf(io.Reader(os.Stdin)))
-		case "Time", "GetCertificate", "GetConfigForClient", "VerifyPeerCertificate", "VerifyConnection", "GetClientCertificate", "ServerECHProvider", "CFEventHandler":
+		case "Time", "GetCertificate", "GetConfigForClient", "VerifyPeerCertificate", "VerifyConnection", "GetClientCertificate", "GetDelegatedCredential", "ServerECHProvider", "CFEventHandler":
 			// DeepEqual can't compare functions. If you add a
```

Comment:
I don't think we added it to test clone func fields

Reply:
I have no opposition to this, so I'll just remove it ;)

Reply:
Ah, no. The testing suite actually complains:

```
tls_test.go:839: all fields must be accounted for, but saw unknown field "GetDelegatedCredential"
```

Reply:
Yeah, you should add it to the other test function TestCloneFuncFields according to the comment.

Reply:
I added to the `TestCloneFuncFields` func but it still needs to be listed (but not set) on `TestNonCloneFuncFields`.